test/test_projective.jl

```diff
@@ -34,12 +34,21 @@ gpt2 = [100.0, -200.0, 0.0]
 (xy, visible) = cameraproject(Cam, [gpt1 gpt2], computevisibility=true)  # project to image
 
 # Then project image points to ground plane to see if we reconstruct them
-# ** Note this does not seem to work with lens distortion
 planeP = [0.0,0.0,0.0]
 planeN = [0.0,0.0,1.0]
 planept = imagept2plane(Cam, xy, planeP, planeN)
 @test maximum(abs.(planept - [gpt1 gpt2])) < f*tol
 
+#same test as above with radial distortion
+dCam=deepcopy(Cam)
```

Comment:
```suggestion
dCam = copy(Cam)
```

Reply:
I think `copy` is not defined for the camera structure. I used deepcopy as simple workaround. But maybe it is misusing the `deepcopy` method here, so we could just create a new camera instance here from parameters, maybe?

Reply:
Since it's test-related codes, it doesn't matter much. But for source codes in `src/`, usage of `deepcopy` should be carefully revisited because it's not well-defined and usually not performant.